dace/codegen/targets/fpga_helper/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
+from .fpga_utils import (
+    is_hbm_array,
+    iterate_hbm_multibank_arrays,
+    modify_distributed_subset,
+    get_multibank_ranges_from_subset,
+    parse_location_bank,
+    ptr,
+)
```

Comment:
Why does this need a separate folder and files? Can't we just put all of this in `fpga.py`?